src/MongoDB.Driver/ClientEncryption.cs

```diff
@@ -110,23 +113,24 @@ public BsonBinaryData Encrypt(BsonValue value, EncryptOptions encryptOptions, Ca
         /// <returns>The encrypted value.</returns>
         public async Task<BsonBinaryData> EncryptAsync(BsonValue value, EncryptOptions encryptOptions, CancellationToken cancellationToken)
         {
-            var bytes = value.ToBson(); //todo:
+            byte[] bytes = new BsonDocument("v", value).ToBson(serializer: BsonValueSerializer.Instance);
```

Comment:
The "v" key is required since otherwise, the server throws an error. Also, this is how it's implemented in Java